### ISSUE TITLE
[UpWork] Fix MediaInfo

### ISSUE THREAD

**Issue by rix1337** [rix1337/docker-verarbeitung#1]
As you can see, the container relies on a (working) bash script that uses filebot to name and replace files.
To gather video resolution for renaming, filebot uses a (third party) package called MediaInfo, which I tried to install in various ways, e.g.:
https://github.com/rix1337/docker-verarbeitung/tree/1624f63e9f39f5c84c563e9c34911bd1b0d1903b

Your task is to fix the current error when running:
docker exec -it Verarbeitung /filebot/filebot.sh -script fn:sysinfo

MediaInfo: net.filebot.mediainfo.MediaInfoException: Unable to load amd64 (64-bit) native library libmediainfo.so: Unable to load library 'zen': Native library (linux-x86-64/libzen.so) not found in resource path (file:/opt/filebot/FileBot.jar

In other words. Filebot has to pick up video resolution again. In my latest commit I have tried placing pre-build official MediaInfo Libraries in the filebot folder, to no success.

This would help me greatly staying with alpine!

Further links:
https://github.com/jirutka/alpine-aports/blob/master/testing/libzen/
https://github.com/jirutka/alpine-aports/blob/master/testing/libmediainfo/

those two should also help setting up mediainfo
